src/components/Loading/LoadingMessage/__tests__/LoadingMessage.spec.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * @file Loading message tests.
+ * @copyright IBM Security 2020
+ */
+
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { LoadingMessage } from '../../../..';
+
+import { carbonPrefix } from '../../../../globals/namespace';
+
+describe('LoadingMessage', () => {
+  test('should have no Axe or DAP violations with overlay', async () => {
+    const main = document.createElement('main');
+    render(
+      <LoadingMessage active withOverlay>
+        test message
+      </LoadingMessage>,
+      {
+        // DAP requires a landmark '<main>' in the DOM:
+        container: document.body.appendChild(main),
+      }
+    );
+    await expect(document.body).toHaveNoAxeViolations();
+    await expect(document.body).toHaveNoDAPViolations(
+      'LoadingMessage with overlay'
+    );
+  });
+
+  test('should have no Axe or DAP violations without overlay', async () => {
+    const main = document.createElement('main');
+    render(
+      <LoadingMessage active withOverlay={false}>
+        test message
+      </LoadingMessage>,
+      {
+        // DAP requires a landmark '<main>' in the DOM:
+        container: document.body.appendChild(main),
+      }
+    );
+    await expect(document.body).toHaveNoAxeViolations();
+    await expect(document.body).toHaveNoDAPViolations(
+      'LoadingMessage without overlay'
+    );
+  });
+
+  test('should have no Axe or DAP violations when inactive', async () => {
+    const main = document.createElement('main');
+    render(<LoadingMessage active={false}>test message</LoadingMessage>, {
+      // DAP requires a landmark '<main>' in the DOM:
+      container: document.body.appendChild(main),
+    });
+    await expect(document.body).toHaveNoAxeViolations();
+    await expect(document.body).toHaveNoDAPViolations(
+      'LoadingMessage when inactive'
+    );
+  });
+
+  test('should add custom class', () => {
+    const { getByTestId } = render(
+      <LoadingMessage data-testid="test-id" className="custom-class" />
+    );
+    expect(getByTestId('test-id').parentNode).toHaveClass('custom-class');
+  });
+
+  test('should pass extra props through spread attribute', () => {
+    const { queryByLabelText } = render(
+      // `description` provides a custom label element
+      // to the inner `Loading` component:
+      <LoadingMessage description="test description" />
+    );
+    expect(queryByLabelText('test description')).toBeInTheDocument();
+  });
+
+  test('should show a custom loading message via `children`', () => {
+    const { queryByText } = render(
+      <LoadingMessage>test message</LoadingMessage>
+    );
+    expect(queryByText(/test message/i)).toBeVisible();
+  });
+
+  test('should apply a custom class', () => {
```

Comment:
It looks like you have the same tests here: https://github.com/carbon-design-system/ibm-security/pull/369/files#diff-6bca8213755a4e7a871fd06f980454edR60-R65

I like this one better 😄